src/meson.build

```diff
@@ -250,3 +250,13 @@ foreach opt: opt_src
         aegisub_src += files(opt[1])
     endif
 endforeach
+
+conf_h = configuration_data()
+if cc.has_function('alloca', prefix: '#include <stdlib.h>')
+    # We already use <stdlib.h> unconditionally.
```

Comment:
If you do end up using meson checks, this comment would need to be clarified (e.g. "The files using alloca already include <stdlib.h> unconditionally")